datacube/model/__init__.py

```diff
@@ -367,22 +325,22 @@ def __init__(self, **kwargs):
 
         super().__init__(**kwargs)
 
-    def __getattr__(self, key):
+    def __getattr__(self, key: str) -> Any:
         """ Allow access to items as attributes. """
         v = self.get(key, self)
         if v is self:
             raise AttributeError("'Measurement' object has no attribute '{}'".format(key))
         return v
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return "Measurement({})".format(super(Measurement, self).__repr__())
 
-    def copy(self):
+    def copy(self) -> 'Measurement':
```

Comment:
Is this supposed to be `'Measurement'` or `Measurement`?

Reply:
@omad this needs to be a string due to mypy limitation: any class that is defined later in the same file has to be done that way, including "self" class. Many of our files have this issue as we define `HigherLevel` class first with `LowerLevel` implementation classes below, mypy prefers "define before use" pattern. In this case it will always be string, because class is not defined until last method is finished.